frontend/src/app/faq/reducers/faq.reducer.ts

```diff
@@ -15,8 +15,21 @@ export function reducer(state: FaqsState = initialState, action: faqActions.Acti
     case FAQS_LOADED:
          return {
         ...state,
-        faqs: action.payload as FaqType[]
+        faqs: action.payload as Faq[]
       };
+
+    case ADD_FAQ:
+      return {
+        ...state,
+        faqs: [...state.faqs, action.payload as Faq]
+      }
+
+    case REMOVE_FAQ:
+      return {
+        ...state,
+        faqs: [...state.faqs.filter(faq => faq !== <Faq>action.payload)  ]
```

Comment:
use property value against object reference, 
BEST solution that add ID property to Faq interface and use that, because the database object must have some kind of ID in the future